modules/data-loading/pages/data-loading-ha.adoc

```diff
@@ -0,0 +1,25 @@
+= Data Loading High Availability (HA)
+:page-aliases:
+
+Data Loading High Availability (HA) provides load balancing and automatic fail over in the event of a service disruption.
```

Comment:
Need to give a notice that this is a beta feature and can cause a performance regression.

Reply:
It no longer causes performance regression. I will update it is a beta version.